locales-en-US.xml

```diff
@@ -99,6 +131,50 @@
     <term name="long-ordinal-10">tenth</term>
 
     <!-- LONG LOCATOR FORMS -->
+    <term name="act">			 
+      <single>act</single>
+      <multiple>acts</multiple>						 
+    </term>
+    <term name="appendix">			 
+      <single>appendix</single>
+      <multiple>appendices</multiple>						 
+    </term>
+    <term name="article-locator">			 
+      <single>article</single>
+      <multiple>articles</multiple>						 
+    </term>
+    <term name="canon">			 
+      <single>canon</single>
+      <multiple>canons</multiple>						 
+    </term>
+    <term name="elocation">			 
```

Comment:
@bwiernik you've added a short form for this. Shouldn't we then also add a long form for this?
(OTOH, I'm a bit unsure about this having a term at all. Can't this just stay empty, just as timestamps?)

Reply:
@adam3smith What do you think? I've mostly seen kindle editions, etc. cited with the term "location" when these are cited (versus page numbers)

Reply:
Ok. In that case I'd say we add "location/locations" as the long form. (We shouldn't be to strict here anyway as styles can always override the defaults. And if something doesn't make sense people can always complain and we make reasonable changes later. But for this, folks need a chance to test use this.)